src/app/pages/animal/register-community-animal-by-census/register-community-animal-by-census.page.html

```diff
@@ -0,0 +1,221 @@
+<app-general-container-component>
+  <h1>Registrar animales comunitarios</h1>
+
+  <mat-form-field appearance="outline" class="w-100">
+    <mat-label>Buscar por cédula</mat-label>
+    <input matInput [formControl]="cedulaControl" />
+  </mat-form-field>
+
+  <button mat-raised-button color="primary" (click)="onSearchByCedula()">
```

Comment:
El atributo color ya no sirve para nada en Material 3

```suggestion
  <button mat-raised-button (click)="onSearchByCedula()">
```